lib/xing/serializers/paged_data.rb

```diff
@@ -0,0 +1,55 @@
+require 'xing/serializers/base'
+
+module Xing::Serializers
+  class PagedData < Base
```

Comment:
I would suggest naming this PagedList for consistency with our usage of List elsewhere.  But, we now have a solid deprecation mechanism for this gem so if you'd rather not do that now I think we can save that for later.